src/OptionParser.php

```diff
@@ -94,15 +94,14 @@ protected function preprocessingArguments(array $argv)
     {
         // preprocessing arguments
         $newArgv = array();
-        $extra = array();
         $afterDash = false;
         foreach ($argv as $arg) {
             if ($arg === '--') {
                 $afterDash = true;
                 continue;
```

Comment:
The `continue` loop should be removed. And the `--` will be added in the `$newArgv` array.